packages/security.rb

```diff
@@ -0,0 +1,15 @@
+package :security do
+  description 'Security features: fail2ban daemon, iptables rules'
+  
+  apt 'fail2ban iptables-persistent' do
+    post :install, [
+      'sudo cp /etc/fail2ban/jail.conf /etc/fail2ban/jail.local',
+      "sed -i 's/bantime  = 600/bantime = 86400/g' /etc/fail2ban/jail.local",
+      '/etc/init.d/fail2ban restart'
```

Comment:
btw, best practice to prefer upstart `service` cmd instead of direct invoke `/etc/init.d` scripts.